scripts/qesap/lib/cmds.py

```diff
@@ -371,9 +371,15 @@ def cmd_ansible(configure_data, base_project, dryrun, verbose, destroy=False, pr
         if dryrun:
             print(' '.join(command['cmd']))
         else:
-            ret, _ = lib.process_manager.subprocess_run(**command)
+            ret, out = lib.process_manager.subprocess_run(**command)
             log.debug("Ansible process return ret:%d", ret)
+            playbook_path = command['cmd'][4]
```

Comment:
does a selection as simple as "5th element" always works to get the playbook name? Is it not the presence or absence of `-vvv` enough to break this mechanism? Is it covered by UT?

If there's a motivation that we are aware that will make it always work we should put it as code comment (I'll do the same in the terraform function later in a next PR)

Reply:
How this works for the first command that is `ansible` and not `ansible-playbook` so does not have any playbook file?

Reply:
Yes have a file for ansible command and it is called ansible.all.log.txt